src/Payone/Webhook/Handler/AutoCaptureHandler.php

```diff
@@ -30,11 +31,25 @@ public function process(SalesChannelContext $salesChannelContext, Request $reque
             return;
         }
 
-        $this->automaticCaptureService->captureIfPossible($paymentTransaction, $salesChannelContext);
+        $txAction = $request->request->get('txaction');
```

Comment:
is there a reason, why you didn't add this code-block to automaticCaptureService?
please move this code-block and the tests to the scope of the automaticCaptureService. 

so the units are clearer.

Reply:
the code is fine for me.

Reply:
I see it differently. If I were to move it to the captureIfPossible function, I would have to pass the request or at least the txAction to the function and then the function gets a dependency on the request. But I don't see it as part of the function to make something dependent on the request. This should remain part of the WebhookHandler.

Reply:
ah i see. But i think this is totally fine. 

i also wont make it depend on the request, but on the tx-action. I am totally fine with it, if action would be an optional argument.

Reply:
I wouldn't even make it dependent on the tx-action. The function can theoretically also be called from a completely different code location (command, ...), so why make it dependent on the webhook?

Reply:
and this is the reason why I said to make the argument optional, so it won't be depend on the webhook

Reply:
But then I don't understand the point behind it. It makes more sense to make the webhook-specific check in the WebhookHandler than to move it as an optional check to the captureIfPossible function, which should actually be independent of the webhook.